pegasus/router.rb

```diff
@@ -408,6 +408,7 @@ def render_template(path, locals={})
       end
       raise e
     rescue => e
+      Honeybadger.context({path: path, e: e})
```

Comment:
Oops, it wasn't part of the main block of `require`s, so I missed it seeing it. Ignore me, LGTM.